### PR TITLE
CIN-06064 - Removed URL param logic during login that is now handled by the Angular SDK

### PR DESCRIPTION
**Pull request by bibincinchy**
CIN-06064 - Removed URL param logic during login that is now handled by the Angular SDK

### CODE REVIEW

**Review by DAquilinaCinchy — CHANGES_REQUESTED** [review]
In the CinchyDynamicFormsComponent, there is a condition on line 310 that was included in the case of a queryParam/session mismatch. Please correct that condition and confirm that the behaviour of the loadForm function remains consistent.

Current path:

```
if (this.rowId) {
  if (!selectedLookupRecord) {
    // deselect record
  }
  else {
    // load record
  }
}
```

Correct:

```
if (selectedLookupRecord) {
  // load record
}
```

File: src/app/app.component.ts
[the remark this answers is not in the document]
Comment (bibincinchy) [reply src/app/app.component.ts:1]: done

File: src/app/app.component.ts [hunk 1]
```diff
@@ -155,8 +97,30 @@ export class AppComponent implements OnDestroy, OnInit {
...
+  
+    if (uri) {
+      idAsString = this.getQueryStringValue("rowId", uri);
+    }
+  
+    if (idAsString) {
+      try {
+        idAsNumber = parseInt(idAsString);
+  
+      return idAsNumber;
```
Comment (DAquilinaCinchy) [inline src/app/app.component.ts:117]: Please indent this by one more level

**Review by DAquilinaCinchy — DISMISSED** [review]

File: src/app/dynamic-forms/cinchy-dynamic-forms.component.ts [hunk 2]
```diff
@@ -307,39 +307,35 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges {
               await this._formHelperService.fillWithFields(form, this.rowId, this.formMetadata, formFieldsMetadata, selectedLookupRecord, tableEntitlements);
 
               // This may occur if the rowId is not provided in the queryParams, but one is available in the session
```
[the remark this answers is not in the document]
Comment (bibincinchy) [reply src/app/dynamic-forms/cinchy-dynamic-forms.component.ts]: done

File: src/app/app.component.ts (hunk 1 above)
Comment (bibincinchy) [reply src/app/app.component.ts]: done

**Pull request merged**